### PR TITLE
Add CI + fix test + update pom + release drafter

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: src/test/resources/org/jvnet/hudson/crypto/site.crt [hunk 1]
```diff
@@ -1,161 +1,32 @@
...
-dvWBtrtiGrpagS8wFwYDVR0gBBAwDjAMBgorBgEEAdZ5AgUBMDAGA1UdHwQpMCcw
-JaAjoCGGH2h0dHA6Ly9wa2kuZ29vZ2xlLmNvbS9HSUFHMi5jcmwwDQYJKoZIhvcN
-AQEFBQADggEBACXU8QZFXBLXVl7VQ4cO3pebKzvLmmgO6+CEEqoucjSbJCh7WOU2
-0DKloSE1CwsM/3fhhN/3qZ4T/VoHFeZ+IRwItDMAWonpVSBMf8UkBZQk5bgsmQ3X
-tXAT2qbLYFR1XdkLMUgds3/TTlWGyqEuUFI4q1NRTNpwWTYO9W63/6q4B9e+dkvS
-MnUyx8/9JhoguyFUo8WdQ3Li/HSFYxI2ltPWzLo7zNooP/glLbpIzXh9Et58W0hh
-x3nTIi5GyVpzdweltoenpb77QGq70l+6+/2ajwKRM7kAN+oezN/3hbiWEpG4iFMB
-oA2mna7jpeuZHgUf8ZgBSTy1LFPzJkzxlaU=
------END CERTIFICATE-----
+MIIFizCCBHOgAwIBAgIRALgy9sW+NYwCCAAAAAAyCqkwDQYJKoZIhvcNAQELBQAw
```
Comment (jglick) [inline src/test/resources/org/jvnet/hudson/crypto/site.crt:2]: Why?

Comment (timja) [reply src/test/resources/org/jvnet/hudson/crypto/site.crt:2]: https://github.com/jenkinsci/lib-crypto-util/blob/4689a2ca2fd0a2874782dba6c2dfa80ea7400b57/src/test/java/org/jvnet/hudson/crypto/PKIXTest.java#L23

**Comment by timja** [comment]
> Fine with me, thanks! Note that `buildPlugin()` will most likely fail for this component (HPI publishing), but we can fix it in a follow-up. I should finally finish [jenkins-infra/pipeline-library#56](https://github.com/jenkins-infra/pipeline-library/pull/56) for the record
> 
> Will merge and configure ci.jenkins.io

should work, we use it in plugin-installation-manager-tool here:
https://github.com/jenkinsci/plugin-installation-manager-tool/blob/master/Jenkinsfile#L4

**Comment by oleg-nenashev** [comment]
Thanks @timja for doing it!
CI setup should be up and running: https://ci.jenkins.io/job/jenkinsci-libraries/job/lib-crypto-util/

**Pull request merged**